requirements.txt

```diff
@@ -6,6 +6,7 @@ google_api_python_client>=1.6.7
 google_auth_oauthlib>=0.2.0
 googleads>=14.1.0
 isort==4.3.9
+lxml<4.3;python_version<"3.6"
```

Comment:
Why not `lxml<4.3;python_version>="3.5"`?

Reply:
for Python 3.6 and 3.7 higher versions of lxml work fine so there is no need to downgrade them. lxml is automatically installed later as dependency if it doesn't exist.

Reply:
I get it, thank you !